game/src/edit/roads.rs

```diff
@@ -421,21 +426,36 @@ fn make_main_panel(
         let stack_batch = stack.batch();
         let stack_bounds = stack_batch.get_bounds();
 
+        let mut rounding = CornerRadii::zero();
+        if idx == 0 {
+            rounding.top_left = DEFAULT_CORNER_RADIUS;
+            rounding.bottom_left = DEFAULT_CORNER_RADIUS;
+        }
+        if idx == lanes_len - 1 {
+            rounding.top_right = DEFAULT_CORNER_RADIUS;
+            rounding.bottom_right = DEFAULT_CORNER_RADIUS;
+        }
+
         current_lanes_ltr.push(
             ctx.style()
                 .btn_plain
                 .btn()
                 .image_batch(stack_batch, stack_bounds)
                 .disabled(Some(id) == current_lane)
                 .bg_color(ctx.style().section_bg, ControlState::Default)
+                .bg_color(ctx.style().section_bg.shade(0.1), ControlState::Hovered)
                 .bg_color(ctx.style().btn_solid_primary.bg, ControlState::Disabled)
+                .image_color(ctx.style().btn_plain.fg, ControlState::Disabled)
                 .image_dims(60.0)
-                .outline(ctx.style().btn_outline.outline, ControlState::Disabled)
+                .padding_top(32.0)
+                .padding_bottom(32.0)
+                .corner_rounding(rounding)
                 .build_widget(ctx, format!("modify {}", id)),
         );
     }
-    // Disable the default margin between buttons
-    let current_lanes_ltr = Widget::custom_row(current_lanes_ltr);
+
+    let current_lanes_ltr =
+        Widget::evenly_spaced_row(current_lanes_ltr, 2).bg(Color::hex("#979797"));
```

Comment:
Maybe not the ultimate solution, but this is an easy way to have "dividers" show up between the lanes.

Reply:
Very clever, I love it!